keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -693,8 +1311,200 @@ type NamedResourcesStringSlice struct {
 All names must be DNS sub-domains. This excludes the "/" character, therefore
 combining different names with that separator to form an ID is valid.
 
+In the Go types above, all structs starting with `NamedResources` are part of
+that structured model. Code generators (more specifically, the applyconfig
+generator) assume that all Go types of an API are defined in the same Go
+package. If it wasn't for that, defining those structs in their own package
+without the `NamedResources` prefix would be possible and make the Go code
+cleaner without affecting the Kubernetes API.
+
+##### ResourceClass
+
+```go
+// ResourceClass is used by administrators to influence how resources
+// are allocated.
+//
+// This is an alpha type and requires enabling the DynamicResourceAllocation
+// feature gate.
+type ResourceClass struct {
+    metav1.TypeMeta
+    // Standard object metadata
+    // +optional
+    metav1.ObjectMeta
+
+    // DriverName defines the name of the dynamic resource driver that is
+    // used for allocation of a ResourceClaim that uses this class.
+    //
+    // Resource drivers have a unique name in forward domain order
+    // (acme.example.com).
+    DriverName string
```

Comment:
Actually, we don't need this anymore for structured parameters. As the description says, this determines which control plane controller to use, but there is none.

Unless someone can think of some use for this, I'll move it to the other KEP.